### PR TITLE
Remove //base dependency from engine.cc

### PR DESCRIPTION
**Pull request by abarth**
We now just call stat directly to test whether a file exists.

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde

File: sky/shell/ui/engine.cc [hunk 1]
```diff
@@ -29,6 +30,23 @@ namespace sky {
 namespace shell {
 namespace {
 
+bool PathExists(const std::string& path) {
+  struct stat info;
+  return stat(path.c_str(), &info) == 0;
```
Comment (chinmaygarde) [inline sky/shell/ui/engine.cc]: If I am not going to use the stat struct, I prefer access. This if fine too.

**Comment by chinmaygarde** [comment]
lgtm

File: sky/shell/ui/engine.cc (hunk 1 above)
Comment (abarth) [inline sky/shell/ui/engine.cc]: Neat.

**Pull request merged**